lib/tasks/tasks.rake

```diff
@@ -788,3 +788,11 @@ namespace :terminology do |_argv|
     Inferno::Terminology.create_validators(validator_type)
   end
 end
+
+namespace :generator do |_argv|
+  desc 'Generate US Core R4 Tests'
+  task :us_core_r4 do
+    path = File.expand_path('../../generators/uscore-r4/generator.rb', __dir__)
+    system('ruby', path)
```

Comment:
This is where it would be really nice if the generator were a class so that you could say something like `Generator.run` rather than having to use `system`.